pkg/controller/controller.go

```diff
@@ -250,7 +253,7 @@ func (p *csiProvisioner) Provision(options controller.VolumeOptions) (*v1.Persis
 			PersistentVolumeReclaimPolicy: options.PersistentVolumeReclaimPolicy,
 			AccessModes:                   options.PVC.Spec.AccessModes,
 			Capacity: v1.ResourceList{
-				v1.ResourceName(v1.ResourceStorage): options.PVC.Spec.Resources.Requests[v1.ResourceName(v1.ResourceStorage)],
+				v1.ResourceName(v1.ResourceStorage): resource.MustParse(repBytesString),
```

Comment:
I think we should not provide this value in bytes. I checked already existing PV and it displays the value rounded up to Gigabytes.

[root@kube-3 ~]# kubectl get pv -o yaml kubernetes-dynamic-pv-8ecdbbbd1e2311e8
apiVersion: v1
kind: PersistentVolume
  capacity:
    storage: 1Gi

Reply:
Ok,  sounds good.